install.sh

```diff
@@ -66,18 +63,20 @@ msg "Currently, you can install one of 'makedeb', 'makedeb-beta', or"
 msg "'makedeb-alpha'."
 msg "If you're not sure which one to choose, pick 'makedeb'."
 
-while true; do
-    read -p "$(question "Which release would you like? ")" response
+if [[ "${MAKEDEB_PKG:-undefined}" == 'undefined' ]]; then
```

Comment:
Can you just do `-x`? I know I'm getting into nitpicky territory, but I pretty much always stick with just `x`, and I'm pretty sure places where `:+` and `:-` are used in the makedeb codebase `x` is used anyway as well.

Reply:
Sorry :) Yeah I'll do 'x' I thought you used it as a placeholder for me to figure out, I'll do 'x', no worries about nitpicking!

Reply:
Also just released that if a user passes in `MAKEDEB_PKG=x` that this would still show the prompt, which probably isn't wanted. We should probably use something like this instead:

```sh
if [[ "${MAKEDEB_PKG:+x}" == '' ]]; then
```

Also it doesn't look like the check to see if the package is valid is ran when the variable is defined. Maybe the logic for that stuff could be put into a function, and then it can be ran in the package chooser as well as when the package is predefined?

Reply:
Sorry for all those changes :P. I think after that last stuff it should be good though

Reply:
Absolutely no need to be sorry :) appreciate the input, pushed a new version.